### ISSUE TITLE
Burn In

### ISSUE THREAD

**Issue by harry-uglow** [k-r-allen/residual-policy-learning#12]
I'm using Residual RL on a current project and I'm interested in "burning in" the critic as you do in your paper. I was looking for more details on how this works but can't find reference to it in the code. 

To burn-in do I simply interact with the environment and train as normal, apart from not updating the actor until the critic's loss is low enough?

Thanks in advance!

**Comment by k-r-allen** [comment]
You should be able to take a look at "train_staged.py" here: https://github.com/k-r-allen/residual-policy-learning/blob/master/tensorflow/experiment/train_staged.py#L68-L70

**Comment by harry-uglow** [comment]
Thanks for pointing that out 😄

**Comment by rhshi** [comment]
I have a few questions on this as well. What exactly is skip training doing? The default is that it is True, so regardless of the burn in the model seems to not train at all. Also, the critic should be "pre-trained," correct? However, the way it is set up seems to be that it can always set the policy learning rate to 0 - during training, maybe the critic suddenly shows poorer performance, and so it is possible that this if statement is true. Lastly, why take the difference in losses/it seems that the actor loss is what is actually being pre-trained (you have losses.append(actor_loss) in the training loop).  Thanks!
